src/test/TxTests.cpp

```diff
@@ -114,11 +114,12 @@ applyCheck(TransactionFramePtr tx, Application& app, bool checkSeqNum)
 
         REQUIRE((!res || tx->getResultCode() == txSUCCESS));
 
+        // this is not true for protocol versions < 10
```

Comment:
This is not the case when we execute setSigner and setThresholds separately. It can be put in better commit after rebasing.

Reply:
cool, instead of disabling the check like you did in a later commit, just add the proper condition in the code: most operations should fail the same way. I can see how certain early failures (transaction level, like auth) could fail differently - but those are the exceptions to the rule :)

Reply:
FYI - this still needs to be changed (and not blanket exempt protocol versions less than 10)